frontend/src/services/chatService.ts

```diff
@@ -1,8 +1,11 @@
 import ActionType from "#/types/ActionType";
 import Session from "./session";
 
-export function sendChatMessage(message: string): void {
-  const event = { action: ActionType.MESSAGE, args: { content: message } };
+export function sendChatMessage(message: string, agentType: string): void {
```

Comment:
Not sure what mentabot is trying to achieve with the **UI** changes here.
Imo, none of them would be merged.

Reply:
Yup, that's what they like to do sometimes, can't take their word for it. 😂